documentation/mDNS_Support.md

```diff
@@ -20,17 +20,18 @@ Our sample file `physical-web-url.service` looks like this:
 <!DOCTYPE service-group SYSTEM "avahi-service.dtd">
 
 <service-group>
-  <name replace-wildcards="yes">http://www.mycompany.com/xyz.html</name>
+  <name replace-wildcards="yes">Physical Web on %h</name>
   <service>
-    <host-name>www.mycompany.com</host-name>
-    <type>_http._tcp</type>
+    <type>_urlbookmark._tcp</type>
```

Comment:
Don't know, because I don't know what you want to do with this project.
But if the publisher of the service (or let's they the target field of the DNS SD RR) does not fit the host, you cannot use it.
This has been pointed by the discussion before: http://www.dns-sd.org/txtrecords.html#http